src/org/parosproxy/paros/core/scanner/Alert.java

```diff
@@ -185,13 +186,24 @@ public void setRiskConfidence(int risk, int confidence) {
 		this.risk = risk;
 		this.confidence = confidence;
 	}
-	
+	/**
+	 * @deprecated (TODO add version) Replaced by {@link #setName}.
+	 * Use of alert has been deprecated in favour of using name.
+	 */
+	@Deprecated
 	public void setAlert(String alert) {
-	    if (alert == null) return;
-	    // ZAP: Changed to not create a new String.
-	    this.alert = alert;
+	    setName(alert);
+	}
+	/**
+	 * @since TODO add version

```

Comment:
The description of the method should be before the JavaDoc tags, otherwise it's included in the tag (the since tag can be added after the param/return tags).